dask/dataframe/groupby.py

```diff
@@ -1792,6 +1792,17 @@ def __init__(self, df, by=None, slice=None, **kwargs):
 
     @derived_from(pd.core.groupby.SeriesGroupBy)
     def nunique(self, split_every=None, split_out=1):
+        """
+        Examples
+        --------
+        
```

Comment:
Looks like you've failed the linting checks due to trailing whitespace here: https://travis-ci.org/github/dask/dask/jobs/707841008

```suggestion

```